test/views/MovementConfirmationViewSpec.scala

```diff
@@ -35,8 +35,8 @@ class MovementConfirmationViewSpec extends UnitViewSpec {
 
       messages("movement.arrival.confirmation") mustBe "Arrival has been submitted"
       messages("movement.departure.confirmation") mustBe "Departure has been submitted"
-      messages("movement.choice.arrival.label") mustBe "Arrival"
-      messages("movement.choice.departure.label") mustBe "Departure"
+      messages("movement.choice.arrival.label") mustBe "Arrive a consignment"
+      messages("movement.choice.departure.label") mustBe "Depart a consignment"
```

Comment:
Nit: You can change it to use `ViewMatchers` and `haveTranslation` matcher.

Reply:
Yes, but I prefer to avoid unnecessary changes during this ticket. I'll touch every view and every view on movements frontend use app and has old implementation.